subscriptionform/templates/home/menu.html

```diff
@@ -3,6 +3,18 @@
 
 {% block content %}
     <h2>{% blocktrans %}Welcome!{% endblocktrans %}</h2>
-    <p>{% blocktrans %}If you want to go to your registered participants, or add a participant, click {% endblocktrans %}
-    <a href="{% url 'participants' %}">{% blocktrans %}here{% endblocktrans %}</a>.</p>
+        <p>{% blocktrans %}If you want to go to your registered participants, or add a participant, click {% endblocktrans %}
+            <a href="{% url 'participants' %}">{% blocktrans %}here{% endblocktrans %}</a>.
```

Comment:
je kan dit binnen de blocktrans houden, is wel zo mooi als de zinsconstructie in een andere taal anders moet lopen:

http://stackoverflow.com/questions/3397218/translating-text-blocks-with-django-what-to-do-with-the-html